mrbelvedereci/plan/handlers.py

```diff
@@ -2,8 +2,31 @@
 from django.conf import settings
 from django.db.models.signals import post_save
 from django.dispatch import receiver
+from django.contrib.admin.models import LogEntry, ADDITION
+from django.contrib.contenttypes.models import ContentType
 from github3 import login
 from mrbelvedereci.plan.models import Plan
+from mrbelvedereci.build.models import Build
+
+
+@receiver(post_save, sender=Build)
+def log_manual_build(sender, **kwargs):
```

Comment:
Probably not necessary since we are already getting this info in the objects. Any added benefit?